lib/improver/tests/orographic_enhancement/test_OrographicEnhancement.py

```diff
@@ -413,9 +413,7 @@ def setUp(self):
         self.plugin.humidity = iris.cube.Cube(
             humidity, long_name="relhumidity", units="1",
             dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])
-        self.plugin.svp = iris.cube.Cube(
-            svp, long_name="saturation_vapour_pressure", units="Pa",
-            dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])
+        self.plugin.svp = svp
```

Comment:
Just set this above - no need to declare `svp` alone which is then never used.

Reply:
It has to be done after the declaration of 
`self.plugin = Orographic...`
So instead I have bought the array down.

```python
        self.plugin = OrographicEnhancement()
        self.plugin.temperature = iris.cube.Cube(
            temperature, long_name="temperature", units="kelvin",
            dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])
        self.plugin.humidity = iris.cube.Cube(
            humidity, long_name="relhumidity", units="1",
            dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])

        self.plugin.svp = np.array([[813.6, 878.0, 848.3],
                                    [903.2, 890.5, 922.2],
                                    [922.1, 928.4, 967.9]])

        self.plugin.vgradz = np.array([[0.02, 0.08, 0.2],
                                       [-0.06, 0.12, 0.22],
                                       [0.08, 0.16, 0.23]])
```